index.html

```diff
@@ -166,7 +168,7 @@ <h3>Profesional Art Printing Data</h3>
          </ul>
       </div>
       </div>
-      <button type="submit" class="card_bttn">See Project</button>
+      <button type="submit" class="card_bttn1">See Project</button>
   </div>
 </div>
```

Comment:
- [ ] Your cards currently lack a hover effect, they are static and nothing happens when I hover over them. The cards hover effect/animation is a requirement and can be seen in the Figma template, please address the issue and implement this animation in your cards.

## Your cards
![2023-03-31_235611](https://user-images.githubusercontent.com/104319462/229252267-10664803-24f3-4b52-88fe-d0f473ef5559.png)

## Figma (no hover)
![2023-03-31_235655](https://user-images.githubusercontent.com/104319462/229252652-6d86a484-1529-4749-881c-57eeab38f884.png)

## Figma (with hover)
![2023-03-31_235639](https://user-images.githubusercontent.com/104319462/229252603-4cdd5e50-4673-4804-9f0d-50cd2b0bffc0.png)